### PR TITLE
Intel build in GH actions

### PR DESCRIPTION
**Pull request by stenczelt**
for #328 

let's see if it passes :)

### CODE REVIEW

**Comment by stenczelt** [comment]
@jameskermode the `make test` part fails with the Intel compiler, for some reason quippy is not recognised by python

**Comment by jameskermode** [comment]
Looks like missing intel libraries. Make sure you link with `ifort`, and might also need to add something to `QUIPPY_LDFLAGS`. I suggest trying to build locally in the same Docker image first to speed up the debug cycle.

**Comment by jameskermode** [comment]
Google suggests you're missing `-liomp5 -lpthread`. These should replace `-lgomp` in `QUIPPY_LDFLAGS` that we have for gfortran.

**Comment by jameskermode** [comment]
Could also be that this line should let the funny Intel `-qopenmp` through as well as `-fopenmp`, but not sure if f2py understands that

https://github.com/libAtoms/QUIP/blob/public/quippy/Makefile#L73

File: .github/workflows/Makefile.inc.intel [hunk 1]
```diff
@@ -0,0 +1,28 @@
+MATH_LINKOPTS=
```
Comment (jameskermode) [inline .github/workflows/Makefile.inc.intel]: Why are maths libraries missing? I think that’s why quippy import is currently failing.

**Comment by stenczelt** [comment]
I guess we can close this one now in favour of #405

**Pull request closed without merging**